packages/core-utils/src/optimism/fees.ts

```diff
@@ -63,3 +63,11 @@ export const zeroesAndOnes = (data: Buffer | string): Array<number> => {
   }
   return [zeros, ones]
 }
+
+// Compute the cost of calldata
+export const calldataCost = (data: Buffer | string): BigNumber => {
```

Comment:
Could you add some basic tests (using known calldata is fine) and a JSDoc string